<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Prefs_TextValidator</class>
 <widget class="QWidget" name="Prefs_TextValidator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Text Validator</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="remSpacesGroupBox">
         <property name="title">
          <string>Remove Spaces</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="remSpacesBegin">
            <property name="text">
             <string>at begining of Paragraphs</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="remSpacesEnd">
            <property name="text">
             <string>at end of Parapgraphs</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="remMultiplySpaces">
            <property name="text">
             <string>multiple spaces</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QCheckBox" name="remSpacesAfter">
              <property name="text">
               <string>after:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="remSpacesAfterChars"/>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="remSpacesBefore">
              <property name="text">
               <string>before:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="remSpacesBeforeChars"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="convertSpaces">
         <property name="text">
          <string>convert fixed spaces to normal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="remEmptyLines">
         <property name="text">
          <string>remove empty lines</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="remHyphenation">
         <property name="text">
          <string>remove manual hyphenation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="remBreaks">
         <property name="text">
          <string>remove line breaks</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="remCharsGroupBox">
         <property name="title">
          <string>Remove Chars</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout1">
            <item>
             <widget class="QLabel" name="charlabel1">
              <property name="text">
               <string>Char:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="char1">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="maxLength">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="before1">
              <property name="text">
               <string>before</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="remCharsBeforeChars1"/>
            </item>
            <item>
             <widget class="QLabel" name="after1">
              <property name="text">
               <string>after</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="remCharsAfterChars1"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout2">
            <item>
             <widget class="QLabel" name="charlabel2">
              <property name="text">
               <string>Char:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="char2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="maxLength">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="before2">
              <property name="text">
               <string>before</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="remCharsBeforeChars2"/>
            </item>
            <item>
             <widget class="QLabel" name="after2">
              <property name="text">
               <string>after</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="remCharsAfterChars2"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout3">
            <item>
             <widget class="QLabel" name="charlabel3">
              <property name="text">
               <string>Char:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="char3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="maxLength">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="before3">
              <property name="text">
               <string>before</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="remCharsBeforeChars3"/>
            </item>
            <item>
             <widget class="QLabel" name="after3">
              <property name="text">
               <string>after</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="remCharsAfterChars3"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout4">
            <item>
             <widget class="QLabel" name="charlabel4">
              <property name="text">
               <string>Char:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="char4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="maxLength">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="before4">
              <property name="text">
               <string>before</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="remCharsBeforeChars4"/>
            </item>
            <item>
             <widget class="QLabel" name="after4">
              <property name="text">
               <string>after</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="remCharsAfterChars4"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout5">
            <item>
             <widget class="QLabel" name="charlabel5">
              <property name="text">
               <string>Char:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="char5">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="maxLength">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="before5">
              <property name="text">
               <string>before</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="remCharsBeforeChars5"/>
            </item>
            <item>
             <widget class="QLabel" name="after5">
              <property name="text">
               <string>after</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="remCharsAfterChars5"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <widget class="QLabel" name="messageLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
